cpp/src/jit/cache.cpp

```diff
@@ -67,8 +68,14 @@ boost::filesystem::path getCacheDir()
   auto kernel_cache_path     = boost::filesystem::path(
     kernel_cache_path_env != nullptr ? kernel_cache_path_env : LIBCUDF_KERNEL_CACHE_PATH);
   if (not kernel_cache_path.empty()) {
```

Comment:
When would `kernel_cache_path` be empty? The only scenario I see is someone setting the env `LIBCUDF_KERNEL_CACHE_PATH` to an empty string to disable caching. Any other cases?

Reply:
The rare scenario where someone unsets the environment variable for HOME.

Reply:
Might be good to add a comment above this line.